protocol/avdl/keybase1/common.avdl

```diff
@@ -345,6 +346,7 @@ protocol Common {
     UnixTime ctime;
   }
 
+  // Used for searching by phone number or email
```

Comment:
:+1: we can reorganize these types if something else comes along that needs different "Identity Visibility enum"